src/pages/CubePlaytestPage.js

```diff
@@ -176,7 +176,7 @@ const StandardDraftCard = () => {
           </LabelRow>
           <LabelRow htmlFor="seats" label="Total Seats" className="mb-0">
             <Input type="select" name="seats" id="seats" defaultValue="8">
-              {rangeOptions(4, 11)}
+              {rangeOptions(4, 17)}
```

Comment:
Should we allow down to 2 seats here?

Reply:
I think that's contingent on the other draft modes being implemented anyway